server/src/services/getEmail.js

```diff
@@ -1,13 +1,13 @@
-import {Datastore} from '@google-cloud/datastore';
+import { Datastore } from '@google-cloud/datastore';
 
 const datastore = new Datastore(
-    {projectId: 'feedzback-343709'},
+  { projectId: 'feedzback-343709' },
 );
-const getEmails = ()=>{
-  const query = datastore.createQuery('inbox');
+const getEmails = () => {
+  const query = datastore.createQuery('feedback');
```

Comment:
```suggestion
  const query = datastore.createQuery('feedzback');
```

Je nommerais la table feedzback pour utiliser toujours le terme feedzback, qu'en pensez vous ?